Configuration/Eras/python/Modifier_run2_ECAL_2017_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+run2_ECAL_2017 =  cms.Modifier()
```

Comment:
exactly, this is why I would leave the flat threshold (even the 2017 300 MeV which now is for EE would be OK). I will put he modifier in Run2_2017